portfolio/src/main/java/com/google/sps/data/ChartDataWithTotalCommentCount.java

```diff
@@ -0,0 +1,13 @@
+package com.google.sps.data;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ChartDataWithTotalCommentCount {
```

Comment:
The class name might now need all information of the member variables. Because if you add some new member fields (e.g. the timestamp of the oldest comment, the timestamp of the latest comment) in the future, the class name "ChartDataWithTotalCommentCount" will need to be updated. Previously, I recommend that the local variable name should reflect the content it contains, e.g. accountIdToCommentCountMap. For the class name, we can just use something like "CommentAnalytics".

Reply:
Thank you for pointing that out, I didn't consider that. I will change this.